lua/alternate-toggler.lua

```diff
@@ -20,10 +20,9 @@ local default_table = {
 
 local user_table = vim.g.at_custom_alternates or {}
 
-vim.tbl_add_reverse_lookup(default_table)
-vim.tbl_add_reverse_lookup(user_table)
-
-local merged_table = vim.tbl_extend("force", default_table, user_table)
+local merged_table_original = vim.tbl_extend("force", default_table, user_table)
+local merged_table = vim.fn.copy(merged_table_original)
+vim.tbl_add_reverse_lookup(merged_table)
```

Comment:
Do we really need these changes here? It seems to me that only changing the setup function to add the reverse lookup should do the trick, or maybe I'm missing something?

Reply:
You're right, that's easier, I've changed the code!